task.sql

```diff
@@ -1,11 +1,29 @@
--- Use our database
-USE ShopDB; 
+USE ShopDB;
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Products (Name, Description, Price, WarehouseAmount)
+    VALUES ("AwersomeProduct", "Product Description", 5, 42);
 
--- Start the transaction 
-START TRANSACTION; 
+INSERT INTO Customers (FirstName, LastName, Email, Address)
+    VALUES ("John", "Dou", "j@dou.ua", "far, far away");
 
--- And some data should be created inside the transaction 
+INSERT INTO Orders (CustomerID, Date) 
+VALUES (1, "2023-01-01");
 
-COMMIT; 
\ No newline at end of file
+SET @OrderID = LAST_INSERT_ID();
+
+START TRANSACTION;
+
+SET @Quantity = 1;
+
+INSERT INTO OrderItems (OrderID, ProductID, Count) 
+VALUES (@OrderID, 1, @Quantity);
+
+UPDATE Products 
+SET WarehouseAmount = WarehouseAmount - @Quantity 
+WHERE ID = 1;
+
+COMMIT;
+
+SELECT * FROM Orders;
+SELECT * FROM OrderItems;
+SELECT * FROM Products;
```

Comment:
Need to remove this SELECTs and add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/rda_task_3_transactions/assets/87720583/4809797b-538e-459e-84a3-149a47cfca7e)